catch/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+# Just point the user at our vendored catch. Nothing else to do here
```

Comment:
Alright, I wonder about having the actual source over in vendor/catch then. We could point out that it's vendored in some other way.